tracing-attributes/src/lib.rs

```diff
@@ -176,6 +176,14 @@ mod expand;
 ///     fn my_method(&self) {}
 /// }
 /// ```
+/// Specifying `follows_from` relationships:
+/// ```
+/// # use tracing_attributes::instrument;
+/// #[instrument(follows_from = causes)]
+/// pub fn my_function(causes: &[tracing::Id]) {
+///     // ...
+/// }
```

Comment:
can we also add an example with a single follows-from span, and/or one with a hard-coded list? i feel like this example doesn't immediately make it obvious that the follows-from list is always treated as a literal of "something implementing `IntoIterator`". which, actually, I think we should document explicitly, as well as adding examples.

Reply:
https://github.com/tokio-rs/tracing/pull/2093/commits/5a6a27dc644d96d44141d11acd479c2610f47092 explicitly documents `follows_from`'s type requirements and adds an example where a single follows-from span is provided.